src/olympia/wsgi.py

```diff
@@ -10,27 +6,11 @@
 from django.core.wsgi import get_wsgi_application
 
 
-log = logging.getLogger('z.startup')
-
-# Remember when mod_wsgi loaded this file so we can track it in nagios.
-wsgi_loaded = datetime.now()
-
-
 # Do validate and activate translations before running the app.
 django.setup()
 django.utils.translation.activate(django.conf.settings.LANGUAGE_CODE)
 
-# This is what mod_wsgi runs.
-django_app = get_wsgi_application()
-
 
-# Normally we could let WSGIHandler run directly, but while we're dark
-# launching, we want to force the script name to be empty so we don't create
-# any /z links through reverse.  This fixes bug 554576.
+# This is referenced in docker/uwsgi.ini: module = olympia.wsgi:application
 def application(env, start_response):
-    if 'HTTP_X_ZEUS_DL_PT' in env:
-        env['SCRIPT_URL'] = env['SCRIPT_NAME'] = ''
-    env['wsgi.loaded'] = wsgi_loaded
```

Comment:
what about the `loaded` view that relies on this env value?  If we're removing the code that sets it should the view be removed at the same time?

Reply:
Wei confirmed that the view was useless but I forgot to remove it, I'll do that now.

Reply:
https://github.com/mozilla/addons-server/pull/18621/commits/136bec8a8552350a3179bd395e6793175dda9427